teamA/leeChanWoo/db/migrations/20221006063429_create_users.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE users (
+		id INT NOT NULL AUTO_INCREMENT,
+		name VARCHAR(50) NOT NULL,
+		email VARCHAR(200) NOT NULL,
+		profile_image VARCHAR(1000) NULL,
+		password VARCHAR(200) NOT NULL,
+		created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+		updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+		PRIMARY KEY(id)
```

Comment:
해당 부분을 다른 migration 파일과 동일하게 들여쓰기 (Indentation)를 수정해주세요!
```suggestion
	id INT NOT NULL AUTO_INCREMENT,
	name VARCHAR(50) NOT NULL,
	email VARCHAR(200) NOT NULL,
	profile_image VARCHAR(1000) NULL,
	password VARCHAR(200) NOT NULL,
	created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
	updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
	PRIMARY KEY(id)
```